docs/turborepo.md

```diff
@@ -0,0 +1,36 @@
+# Turborepo and Telescope
+
+Telescope is configured as a monorepo: a single repository that holds many projects. [Turborepo](https://turborepo.org/) is the tool we have chosen to manage every project within Telescope.
+
+You will probably not have to interact with Turborepo directly but it is worth understanding how it work. The [Turborepo docs](https://turborepo.org/docs) do a good job at explaining its benefits and features.
+
+We have a `package.json` file at the root of telescope that defines the rules in what is called a `"pipeline"`.
+
+The pipeline includes all the Turborepo commands that are available to use.
+
+Example:
+
+```
+{
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"]
+    },
+    "lint": {}
+  }
+}
+```
+
+Using Turborepo:
+
+```
+# To run the build command
+pnpm turbo run build
+
+# To run the lint command
+pnpm turbo run lint
+```
+
+Turborepo will look inside each workspace defined in `pnpm-workspace.yaml` for the specified command and will run it in each workspace.
+
```

Comment:
Could we mention a simple example using the `pipeline` you listed above?

Something like this:

```
As an example, suppose we use the pipeline mentioned above.

When you run `pnpm turbo run lint`, Turborepo will go through all projects found in the `pnpm-workspace.yaml`, and look at the project's `package.json` to find the script that is named as `lint`, and thus will run `lint` for each project that has such script.
```